### ISSUE TITLE
Application Questions

### ISSUE THREAD

**Issue by samdog2727** [va-net/flare#218]
**Is your feature request related to a problem? Please describe.**
Not really. This is just a cool idea for a feature.

**Describe the solution you'd like**
VAs are able to add both long answer and multiple choice questions (somewhat like a google form) that pilots can fill out. The multiple choice questions would be very cool if they could be predetermined correct and wrong answers. This information could be collected into an "Applicant's Summary" showing their email, Grade and violations-landings, percentage of correct multiple choice answers. Using this could help fill in variables in email sent to applicant.

**Describe alternatives you've considered**
Using a different platform for an entrance exam.

**Additional context**
I can provide Thai Group Virtual's entrance exam to show what kind of things would be awesome to see.

**Comment by Velocity23** [comment]
Would I be right in saying there are 2 parts to this request? 

1. Custom questions on the application form
2. Exam type thing

I ask because the former is doable, but the latter may have to come as a plugin.

**Comment by samdog2727** [comment]
Yes. I don't understand what you mean by "exam". Would you mind elaborating please?

**Comment by Velocity23** [comment]
Like a test that people have to take before they can be part of the VA. An entrance exam.

**Comment by samdog2727** [comment]
That would be a very cool feature, especially if Flare intends to combine as many aspects of a VA as possible. So yes, that is what I mean.

**Comment by Velocity23** [comment]
Sounds like more of a plugin thing. I'll transfer it.
